src/test/java/com/lob/client/test/AddressTest.java

```diff
@@ -27,6 +27,9 @@
     @Test
     public void testListAddresses() throws Exception {
         final AddressResponseList addresses = client.getAddresses().get();
+        assertEquals(200, addresses.getStatusCode());
```

Comment:
This is apparently the correct way to do it in Java. The error messages are hardcoded such that the first argument is the expected value and the second one is the one you're testing. Otherwise when the test fails, it'll print something like `java.lang.AssertionError: expected:<500> but was:<200>`